src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccount.fs

```diff
@@ -0,0 +1,76 @@
+﻿module Deciders.BankAccount
+
+open System
+
+type BankAccountOpened =
+    { BankAccountId: Guid
```

Comment:
Defining wrapper types in F# is super cheap:
```
[<Struct>]
type AccountId = AccountId of Guid

module AccountId =
   let newId() = AccountId (Guid.NewGuid())
```
Use it for all ids, and currency code.

Reply:
Good call. Added in https://github.com/oskardudycz/Ogooreck/pull/13/commits/c7166cf28397d6aae2695bdcac35c1c213df453f.